'make' failed with "undefined: Config"

$ make
CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o _output/bin/snapshot-operator cmd/snapshot-operator/main.go
\# github.com/openshift/snapshot-operator/pkg/operator
pkg/operator/deployment.go:36:75: undefined: Config
pkg/operator/handler.go:57:6: undefined: Config
make: *** [Makefile:15: snapshot-operator] Error 2


$ git log -1
commit 249047674ab427edafe15af1508ea24f83604ea4 (HEAD -> master, origin/master, origin/HEAD)
Author: Tomas Smetana <tsmetana@redhat.com>
Date:   Wed Oct 24 13:27:51 2018 +0200
    Update RBAC to work in-cluster

--- a/pkg/operator/handler.go
+++ b/pkg/operator/handler.go
@@ -53,6 +53,12 @@ func NewHandler() sdk.Handler {
 	return handler
 }
 
+type Config struct {
+	cloudProvider      string
+	awsSecretNamespace string
+	awsSecret          string
+}
+
 type Handler struct {
 	cfg Config
 }

